privacy-snapshot.json

```diff
@@ -38,5 +38,6 @@
   "token-api.metaswap.codefi.network",
   "tx-insights.metaswap.codefi.network",
   "unresponsive-rpc.url",
-  "www.4byte.directory"
+  "www.4byte.directory",
+  "price-api.metafi.codefi.network"
```

Comment:
We should be able to remove `api.coingecko.com` from this list now as well

Reply:
Fixed here https://github.com/MetaMask/metamask-extension/pull/22372